source/js/src/utils.js

```diff
@@ -260,14 +259,13 @@ NexT.utils = NexT.$u = {
     var $div = $('<div />').addClass('scrollbar-measure').prependTo('body');
     var div = $div[0];
     var scrollbarWidth = div.offsetWidth - div.clientWidth;
-
     $div.remove();
 
     return scrollbarWidth;
   },
 
   getContentVisibilityHeight: function() {
-    var docHeight = $('#content').height();
```

Comment:
For what needed this changes?

Reply:
So, go see examples with `getContentVisibilityHeight` function. Because i remember, it was work fine (but mb not with all comments systems) and mb not now.

Reply:
You're right. I have think about it, and I think it would be better to change it back... maybe it's just designed like this

Reply:
Actually, this function for rightly back2top percentage ratio. When comments systems exists, horizontal height changed dynamically when page will scrolled, so, b2t percent was counting not rightly.
This all i remember when add this function. For now  some comment systems have dynamic loading (like `lazy` option for Disqus), so, i suggest to retest it in future.

Reply:
Yes, need to discuss `$('#content').height()` and `$('.container').height()` in the future

Reply:
Here is changes for this function & b2t:

1. https://github.com/iissnan/hexo-theme-next/commit/4bd315d0d40ba42a7c6efbb244fa75bef8eb5497
2. https://github.com/iissnan/hexo-theme-next/commit/028fe602f0dd017bc44d9802ed88a0a9a2b6b3be
    and addition https://github.com/iissnan/hexo-theme-next/commit/61086aa23ed8fa2e94580421cbc696852978fc06
    ([PR #1898](https://github.com/iissnan/hexo-theme-next/pull/1898))

So, you can rewiew this.

Reply:
> Yes, need to discuss `$('#content').height()` and `$('.container').height()` in the future

Here is 2 problems:

1. Comment system can be loaded with `lazy` (viewport hight will more only when user scrolled down).
2. Comment system loading from 3rd-party side. So, JS on our server don't know which height we need to count.

But you can try to play around with this. Maybe you got success decision and make this coll feature (or bugfux).

Reply:
What about adding this to NexT v7.x Roadmap? I have reviewed the previous changes, but have no idea how to solve this...

Reply:
You can add it, of course. Anyway, I think NexT have more important things to improvment.

P.S. From next year we will start to work for v7.x and all bugfixes / features will pushed there. We also will resume maintenance for v6.x, but will be pushed onlt bugfixes. I want to create gantt roadmap something like in nodejs (https://github.com/nodejs/Release). Need to think about it better in details, any suggestions are welcome. In ideal need any chat with you to talk about this.